.github/workflows/build.yml

```diff
@@ -0,0 +1,108 @@
+name: Build
+
+on: 
+  push:
+    branches:
+      - 'master'
+  pull_request:
+
+
+jobs:
+  Build:
+    name: Build appimagetool
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        include:
+          - ARCH: 'x86_64'
+            docker-container: 'appimagebuild'
+          - ARCH: 'i386'
+            docker-container: 'appimagebuild-i386'
+          - ARCH: 'armhf'
+            docker-container: 'appimagebuild-armhf-cross'
+          - ARCH: 'aarch64'
+            docker-container: 'appimagebuild-aarch64-cross'
+      fail-fast: false
+    env:
+      DOCKER_IMAGE: quay.io/appimage/${{ matrix.docker-container }}
+      ARCH: ${{ matrix.ARCH }}
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - name: Install dependencies for ARM and AARCH64
+      if: endsWith(matrix.docker-container, 'cross') 
+      run: |
+        sudo apt-get update
+        sudo apt-get -y install binfmt-support qemu-user-static
```

Comment:
I have little confidence this would work. I'll check it against the Travis config ASAP.

Reply:
ok